### ISSUE TITLE
Can't open WebUI UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe3 in position 189: invalid continuation byte

### ISSUE THREAD

**Issue by napalmpt** [binhex/arch-delugevpn#176]
I cannot open webUI, downloads seems to be running normally but if I add a torrent to the watched folder it renames it to invalid.

Also the deluged.log file is getting a lot big! it was 800MB and now  12MB after 12min running. 
It has mostly this errors:
15:17:00.487 [DEBUG   ][deluge_autoadd.core           :267 ] File checked for auto-loading is invalid: xxxx.torrent.added
15:17:00.487 [DEBUG   ][deluge_autoadd.core           :267 ] File checked for auto-loading is invalid: xxxx.torrent.added
15:17:00.487 [DEBUG   ][deluge_autoadd.core           :267 ] File checked for auto-loading is invalid: xxxx.torrent.added

supervisor.log: https://pastebin.com/XmApTujw

**Comment by binhex** [comment]
this looks relevant:-
https://forum.deluge-torrent.org/viewtopic.php?t=55394

**Comment by binhex** [comment]
can you make sure you havent got any plugins enabled, check /config/core.conf to confirm this.

**Comment by napalmpt** [comment]
Magically it started working 5h ago. all I did was testing other vpn servers... now I am in spain, but it failed before too...
The plugins: I have autoadd and execute active. but I only use autoadd. execute: I had nothing there, I disabled it now.
I tried Label plugin a while back but didn't find use for it, it was disabled, but I activated it now, there was a label, but it was simply named "sonarr". I deleted it and disabled plugin.
In the core.conf file I also have simpleextractor plugin there, but it does not appear on the webui, I think I tried to use it a while back but it didn't work.
I still don't know what caused this, maybe somethingg from the VPN provider? Today it was also giving errors about portforward, but after an hour or two no more errors and that's when I got it working fine so far...
Could it be some torrent? I have about 100 seeding and 6 downloading now. I deleted about 30 of the oldest...
